oak_crypto/src/encryptor.rs

```diff
@@ -204,10 +204,18 @@ impl ClientEncryptor {
             .encrypted_message
             .as_ref()
             .context("response doesn't contain encrypted message")?;
+        let nonce = encrypted_message.nonce.clone().try_into().map_err(|_| {
```

Comment:
Could you also print the error, just in case it contains something useful? Also, does `context` not work here?

Reply:
`try_into` returns the original vector, and context is not implemented for `Vec`

Reply:
Also we currently don't print errors in the `oak_crypto` crate. The crate that uses it should print all related errors